src/server/routes/annualData.js

```diff
@@ -146,7 +145,7 @@ function annualListData(req, res, next) {
       next();
     })
     .catch((error) => {
-      logger.error(`Status Code: ${error.statusCode}, Error Message: ${error.code}`);
+      logger.error(`Status Code: ${error.statusCode}, Error Message: ${error.code}, Endpoint: ${platformConfig.endpoints.annualLists[dataType]}, Redirecting to: ${config.baseUrl}/404`, error);
```

Comment:
I think this should also be `source` instead of `endpoint`? Following the same pattern of `annualCurrentListData`?